README.md

```diff
@@ -0,0 +1,27 @@
+# CAOTIC
+High Contrast and Adaptive Optics laboratory testbeds community website
+
+Welcome to the community website for astronomical testbeds! We are currently working on getting our 
+website migrated to this place here, the original website can be found here:  
+https://sites.google.com/view/highcontrastlabs/
+
+## Local setup - MacOS
+
+### Initial setup
+
+1. install homebrew  
+https://osxdaily.com/2018/03/07/how-install-homebrew-mac-os/
```

Comment:
Suggested change: 
```
1. Install hombrew following the instructions on this page: https://osxdaily.com/2018/03/07/how-install-homebrew-mac-os/. Note that sudo privileges are needed in order to get homebrew installed. If you do not have sudo privileges, you will require that someone with those privileges install homebrew for you (in which case, continue with step 2), otherwise, you will not be able to install homebrew
```